lib/debug/server_dap.rb

```diff
@@ -252,27 +252,30 @@ def process
         when 'setFunctionBreakpoints'
           send_response req
         when 'setExceptionBreakpoints'
-          process_filter = ->(filter_id) {
-            case filter_id
-            when 'any'
-              bp = SESSION.add_catch_breakpoint 'Exception'
-            when 'RuntimeError'
-              bp = SESSION.add_catch_breakpoint 'RuntimeError'
-            else
-              bp = nil
-            end
+          process_filter = ->(filter_id, cond = nil) {
+            bp =
+              case filter_id
+              when 'any'
+                SESSION.add_catch_breakpoint 'Exception', cond: cond
+              when 'RuntimeError'
+                SESSION.add_catch_breakpoint 'RuntimeError', cond: cond
+              else
+                nil
+              end
             {
-              verified: bp ? true : false,
+              verified: !bp.nil?,
```

Comment:
bp can be false?

Reply:
I don't think it should be `false` in any case. If it's not added, it should just be `nil`. Why do you expect it to be `false` too?

Reply:
I want to know the reason why the change is needed.

Reply:
It's not a necessary change so if you don't want it I can revert it.